h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -37,7 +37,8 @@
     @API(help="Lambda best + 1 standard error. Only applicable with lambda search and cross-validation")
     double lambda_1se;
 
-    @API(help="Dispersion parameter, only applicable to Tweedie family")
+    @API(help="Dispersion parameter, only applicable to Tweedie family for input/output and fractional Binomial for" +
+            " output only")
```

Comment:
```suggestion
    @API(help = "Dispersion parameter, only applicable to Tweedie family (input/output) and fractional Binomial (output only)")
```

A small tiny suggestion to re-word the description to be more readable. Just my opinion. Feel free not to use it please.